index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+   <title>Alan Turing - Past and Future</title>
+   <link rel="stylesheet" type="text/css" href="css/style.css">
+  </head>
+
+<body>
+  <nav>
+    <strong>Past Achievements</strong>
+     * 
+    <a href="future_goals.html">Future Goals</a>
+  </nav>
+
+  <h1>Alan <strong>M.F.</strong> Turing</h1>
+
+  <hr>
+
+  <h2>Past Achievements</h2>
+
+  <p>
+    Hello, my name is Alan Mathison Turing. I was an English computer scientist, mathematician, logician, cryptanalyst and theoretical biologist who lived in the first half of the 20th century. I was highly influential in the development of theoretical computer science, providing a formalisation of the concepts of algorithm and computation with the Turing machine, which can be considered a model of a general purpose computer. I'm widely considered to be the father of theoretical computer science and artificial intelligence. 
+
+  </p>
+
+  <h3>Career Highlights</h3>
+
+    <ul>
+      <li>
+        <h4>
+          <strong>Cracking Codes and Killing Nazis,</strong> <small>Government Code and Cypher School WWII</small>
+        </h4>
+
+        <p>
+          As one of the UK's leading crypotgraphers, I devised a number of techniques for speeding the breaking of German ciphers, including improvements to the pre-war Polish bombe method, an electromechanical machine that could find settings for the Enigma machine. I played a pivotal role in cracking intercepted coded messages that enabled the Allies to defeat the Nazis in many crucial engagements, including the Battle of the Atlantic; it has been estimated that my work shortened the war in Europe by more than two years and saved over fourteen million lives.
+        </p>
+        <img src="images/turing_400.png"/>
+      </li>
+
+      <li>
+        <h4>
+          <strong>Developer of the Turing Test,</strong> <small>Victoria University Mathematics Department 1948-50</small>
+        </h4>
+
+        <p>
+          In "Computing Machinery and Intelligence", I addressed the problem of artificial intelligence, and proposed an experiment that became known as the Turing test, an attempt to define a standard for a machine to be called "intelligent". My idea was that a computer could be said to "think" if a human interrogator could not tell it apart, through conversation, from a human being.
+        </p>
+        <img src="images/terminator.png"/>
+        <img src="images/bladerunner.png"/>
+      </li>
+    </ul>
+
+    <h3>Education</h3>
+
+    <ul>
+      <li>
+        <strong>Kings College, Cambridge,</strong> <small>First Class Honors, Mathematics, 1931-1934</small> 
+      </li>
+      <li>
+        <strong>Princeton,</strong> <small>Ph.D., Mathematics</small>
+      </li>
+    </ul>
+
+    <hr>
+
+    <h3>Cinematic Inspirations</h3>
+    <nav>
+      <a href="http://www.imdb.com/title/tt0083658/" target="_blank">Bladerunner</a> (a classic movie about machines  mastering the Turing test ... or do they?)
```

Comment:
The common convention for external links is to open them up in a new tab/window.  🔨